lighthouse-cli/bin.js

```diff
@@ -96,12 +96,6 @@ async function begin() {
     cliFlags.outputPath = 'stdout';
   }
 
-  // @ts-expect-error - deprecation message for removed disableDeviceEmulation; can remove warning in v6.
-  if (cliFlags.disableDeviceEmulation) {
```

Comment:
convert this to "--emulated-form-factor has been removed" instead?

Reply:
nevermind I found the config fatal error :)

Reply:
yeah i started that, but then realized this check shoulda been in the config to begin with. ;)